scripts/build/Platform/Windows/build_config.json

```
@@ -449,12 +449,24 @@
         "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo"
     }
   },
+  "install_release": {
```

I think this step should include monolithic as part of its name to describe that it builds _monolithic_ release.
```suggestion
  "install_mono_release": {
```